nifi-nar-bundles/nifi-aws-bundle/nifi-aws-abstract-processors/src/main/java/org/apache/nifi/processors/aws/s3/AbstractS3Processor.java

```diff
@@ -125,16 +125,31 @@
                     new AllowableValue("S3SignerType", "Signature v2"))
             .defaultValue("Default Signature")
             .build();
+
+    public static final PropertyDescriptor CLIENT_SERVICE = new PropertyDescriptor.Builder()
+            .name("Client Service")
```

Comment:
Name should be a machine-readable key like "client-service" with a human-readable displayName like "Client Service".  I know this has not been done previously in this file, but we should try to do it going forward.